hack/lib/serverless.bash

```diff
@@ -3,7 +3,9 @@
 function ensure_serverless_installed {
   logger.info 'Check if Serverless is installed'
   local prev=${1:-false}
-  if oc get knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}" >/dev/null 2>&1; then
+  if oc get knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}" >/dev/null 2>&1&& \
```

Comment:
this looks weird to me: `2>&1&&`

can we group the two && commands with parentheses and apply the redirects to both only once?

Reply:
I simply lack bash skills to give you a good answer but I can give it a try.

Reply:
I just saw the space is somehow deleted.

I think following should be readable enough:
```
if oc get knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}" >/dev/null 2>&1 && \
     oc get knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" >/dev/null 2>&1
```

I will add that space back.